homeassistant/components/homekit/strings.json

```
@@ -0,0 +1,54 @@
+{
+   "title" : "HomeKit Bridge",
+   "options" : {
+      "step" : {
+         "yaml" : {
+            "title" : "Adjust HomeKit Bridge Options",
+            "description" : "This entry is controlled via YAML"
+         },
+         "init" : {
+            "data" : {
+               "include_domains" : "Domains to include"
+            },
+            "description" : "Entities in the “Domains to include” will be bridged to HomeKit. You will be able to select which entities to exclude from this list on the next screen.",            
+            "title" : "Select domains to bridge."
+         },
+         "exclude" : {
+            "data" : {
+               "exclude_entities" : "Entities to exclude"
+            },
+            "description" : "Choose the entities that you do NOT want to be bridged.",
+            "title" : "Exclude entities in selected domains from bridge"
+         },
+         "advanced" : {
+            "data" : {
+               "auto_start" : "Autostart (disable if using Z-Wave or other delayed start system)",
```

Use as much references as possible.

```suggestion
               "auto_start" : "[%key:component::homekit::config::step::user::data::auto_start%]",
```

---

Nice, didn't know that worked :)

---

I'm working on popularizing it. Have a draft pr to migrate bunch of config flow translations. I just have been getting myself into a lot of never ending projects lately, so will get to it soon 🙃